### PR TITLE
Add Blog post on SSWG guides

### PR DESCRIPTION
**Pull request by 0xTim**
Adds a blog post to raise awareness of the SSWG server guides

### CODE REVIEW

**Comment by 0xTim** [comment]
@swift-ci test

File: _posts/2022-08-09-sswg-server-guides.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+date: 2022-08-09 11:00:00
+title: Server Guides now on Swift.org
+author: 0xTim
+---
+
+The [Swift Server Workgroup](/sswg/) has hosted a set of guides for server development on Github for a number of years now. With the open sourcing of the website, we've transitioned the guides to Swift.org! You can find the guides [here](/server/guides/).
+
+The guides cover everything from setting up your environment, building and testing applications, profiling them and deploying them. There are guides for deploying to AWS, both on EC2 and ECS, Digital Ocean, Heroku and GCP. There are also guides for how best to package Docker containers and debug issues in production.
+
+Additionally, the guides contain a section for library authors. These currently cover best practices for logging in libraries at different levels and how to adopt Swift concurrency.
```
Comment (alexandersandberg) [inline _posts/2022-08-09-sswg-server-guides.md:13]: Might be nice to add links here, either to the section or to the mentioned section items.

File: _posts/2022-08-09-sswg-server-guides.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+If you have anything you'd like to see covered in the guides you can either [create an issue](https://github.com/apple/swift-org-website/issues/new/choose) or create a post on the forums.
```
Comment (alexandersandberg) [inline _posts/2022-08-09-sswg-server-guides.md:15]: Maybe a link to the forum section that we suggest they add a post in?

**Review by alexandersandberg — APPROVED** [review]
👍

Left two suggestions.

**Review by cthielen — APPROVED** [review]
Approved with recent commits

**Comment by cthielen** [comment]
@tomerd @0xTim Thursday 8/18 at 1:00 work for me to merge and publish this?

**Comment by 0xTim** [comment]
@cthielen that works for me!

**Comment by cthielen** [comment]
@swift-ci test

**Pull request merged**